src/pages/LandingPage.tsx

```diff
@@ -22,10 +22,10 @@ export const LandingPage: React.FC = () => {
       <Space direction="vertical" size={150}>
         <Row>
           <Col>
-            <div className={styles.LandingPageFullWidthImagesContainer}>
-              <img src={landingPage1} className={styles.ImageStyle} />
+            <div className={styles.LandingPageMapImageContainer}>
+              <LandingPage1 className={styles.MapImageStyle}/>
               <p className={styles.LandingPageOverlayText}>
-                The Mortality Surveillance & Reporting Laboratory at Boston University
+                <strong>Millions</strong> of deaths from COVID-19 are underreported.
```

Comment:
Could you move this into: https://github.com/UncountedDeaths/UncountedDeaths.github.io/blob/react_transfer/src/content/LandingPageContent.ts

You should be able to keep the `<strong>` tag in the string as well. Just name the variable something logical!

Reply:
I tried just exporting the line as a string, but it reads the html tag as a string as well - was there another way you meant?

Reply:
Oh, that's too bad. I'm not sure how to fix that issue then. Just leave it as is for now then!